engine/engine/src/engine_main.cpp

```diff
@@ -117,6 +117,9 @@ int engine_main(int argc, char *argv[])
     {
         glfwAndroidPollEvents();
         dmTime::Sleep(0);
+        if (g_AndroidApp->destroyRequested) {
+            return 0;
```

Comment:
App requested exit. So, let's exit.

Reply:
I think this is a good code comment, so that others in the future can reason about the return statement.